docs/compilers/Visual Basic/Compiler Breaking Changes - VS2015.md

```diff
@@ -108,3 +108,15 @@ According to @AlekseyTs , "Dev10 considers some expressions as invalid in contex
 
 These rules are being relaxed in Roslyn.  A constant expression will be allowed of the form expr.ID where ID is a constant member of the type of the expression.  The expression will not be evaluated and a warning will be issued.  There are no restrictions on the form of expr, other than that it must be syntactically and semantically valid.
 
+### VB Embedded Runtime inherits overflow checking from the compilation
```

Comment:
Since this section is not about a breaking change, does it belong in this file?

Reply:
> Since this section is not about a breaking change, does it belong in this file?

I've moved it to its own file.